harness/features/initialize.cloud.test.ts

```diff
@@ -34,7 +37,10 @@ describe('Client Initialize Tests - Cloud', () => {
                 const client = new CloudTestClient(name)
                 const response = await client.createClient({}, null, true)
                 const body = await response.json()
-                expect(body.exception).toBe('Missing environment key! Call initialize with a valid environment key')
+                expectExceptionMessageToBe(
+                    'Missing environment key! Call initialize with a valid environment key',
```

Comment:
I thought it would make more sense in the context of how I named the function, but I can change the name of the function

Reply:
i think even with this name it makes more sense the other way, since you're saying `expect <exception message> to be <value>`
not `expect <value> to be <exception message>`